docs/source/tutorial_worlds.md

```diff
@@ -13,120 +13,353 @@ section. If you are interested in creating a task, please see
 Introduction
 ------------
 
-This tutorial will cover the details of batched data, and why we use
-Shared Worlds.
+This document discusses the overview of [__Worlds__](parlai.core.worlds.World), a
+core concept within ParlAI. It aims to be a high level overview of
+_how_ some concepts are implemented in ParlAI, so that you can know what's
+happening behind the scenes, but you are unlikely to write any custom worlds
+yourself.
+
+Worlds are where agents live, and the world defines the communication flow
+between agents.  If you are familiar with
+[Reinforcement Learning](https://en.wikipedia.org/wiki/Reinforcement_learning),
```

Comment:
sad we lost the executable world because you need that for proper RL with something like LIGHT... but guess we can always restore when finally get that

Reply:
Yeah I've sketched how it should be for Margaret. But if we're going to add it, it needs to be something that's actually supported -- not test-less code without READMEs or documentation that's 3 years old.